ChromeGaming/Squard-line
# UI Enhancement: Centered game controls with improved start/restart buttons  #145

## PR Description 📜

UI Enhancement: Centered game controls with improved start/restart buttons

This pull request aims to improve the user interface of our game by centring the game controls and enhancing the appearance of the start and restart buttons. These changes will make the game more visually appealing and improve user experience.

Changes implemented:
1. Added a new container div for game controls
2. Centered the start and restart buttons both vertically and horizontally on the page
3. Styled the buttons with distinct appearances:
   - Start button: Green background
   - Restart button: Light grey background with a border
4. Implemented hover effects for better user feedback
5. Adjusted z-index values to ensure buttons appear above the game canvas and background layers

These modifications make the game interface more intuitive and visually cohesive. The centred controls provide better accessibility, especially on different screen sizes.

To test:
1. Load the game and verify that the start and restart buttons are centered
2. Check that the buttons have the correct styling and hover effects
3. Ensure the buttons appear above all other game elements
4. Test on different screen sizes to confirm responsiveness

Screenshots of the changes are attached for reference.

Let me know if any further modifications or clarifications are needed!

<hr>
 
## Mark the task you have completed ✅



- [ ✅] I follow [CONTRIBUTING GUIDELINE](https://github.com/GameSphere-MultiPlayer/Community-Page/blob/main/.github/CONTRIBUTING_GUIDELINE.md) & [CODE OF CONDUCT](https://github.com/GameSphere-MultiPlayer/Community-Page/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [✅ ] I have performed a self-review of my own code or work.
- [ ✅] I have commented my code, particularly in hard-to-understand areas.
- [✅ ] My changes generates no new warnings.
- [✅ ] I have followed proper naming convention showed in [CONTRIBUTING GUIDELINE](https://github.com/GameSphere-MultiPlayer/Community-Page/blob/main/.github/CONTRIBUTING_GUIDELINE.md)
- [✅ ] I have added screenshot for review.


<hr>
## Add your screenshots(Optional) 📸

![image](https://github.com/user-attachments/assets/2d7ea7e7-5334-407e-bf7b-0e099b1aaa9b)





--- 
<br>

## Thank you soo much for contributing to our repository 💗
